app/controllers/api/v1/settings_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class SettingsController < Api::V1::ApiController
+      skip_before_action :authenticate_user!
+
+      def must_update
```

Comment:
This will be consumed only by devices, if they don't want to use this just don't use it. Is not a must check so I don't agree on moving it inside application controller

Reply:
What do you mean by devices? Aren't all clients of the api devices?

Reply:
Android & IOS that consume the API
This is not into scope for web clients that consumes API